pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+line-length = 120
+target-version = ['py35']
+include = '\.pyi?$'
```

Comment:
Is it possible to use the existing `setup.cfg` instead of adding `pyproject.toml`?

Reply:
I think we also need to consider what to do about line length. I know the flake8 config caps at 120, but most of the codebase is a self-enforced less than 90ish.

Reply:
Happy to change line length to your preference, Black's standard is about 90ish, so could 
'just' default to that. 

Note: The project currently has 120 set for flake8 in setup.cfg and Django itself uses 119. 

https://github.com/carltongibson/django-filter/blob/master/setup.cfg#L13

Please confirm and I'll make the change.